src/app/item-popup/PlugTooltip.tsx

```diff
@@ -49,21 +51,64 @@ export default function PlugTooltip({
       )
     : [];
 
+  const stats: { [statHash: string]: number } = {};
+
+  for (const statHash of visibleStats) {
+    const value = plug.stats?.[parseInt(statHash, 10)];
+    if (typeof value === 'number') {
+      stats[statHash] = value;
+    }
+  }
+
+  return (
+    <PlugTooltipContent
+      defs={defs}
+      def={plug.plugDef}
+      perks={plug.perks}
+      stats={stats}
+      plugObjectives={plug.plugObjectives}
+      enableFailReasons={plug.enableFailReasons}
+      cannotCurrentlyRoll={plug.cannotCurrentlyRoll}
+      wishListTip={wishListTip}
+    />
+  );
+}
+
+export function PlugTooltipContent({
```

Comment:
Would be good to document this, explain why its its own thing and when it should be used.

Reply:
Can do.